src/twisted/web/test/test_wsgi.py

```diff
@@ -735,6 +735,16 @@ def channelFactory():
 
         return d
 
+    def test_remotePort(self):
+        """
+        The C{'REMOTE_PORT'} key of the C{environ} C{dict} passed to the
+        application contains the port of the client making the request.
+        """
+        d = self.render("GET", "1.1", [], [""])
+        d.addCallback(self.environKeyEqual("REMOTE_PORT", "12344"))
```

Comment:
I bit hard to read and understand how we endedup with `12344`.

I prefer an explicit test arrangement like in the IPv6 test.

but I think it's ok. it's not worse than what we have in trunk.

this is a minor comment

Reply:
Thanks @adiroiban, I noticed that the `DummyChannel` class in `requesthelper.py` set the port to `12344` by default, but I think being more explicit as you've suggested is always better. I've updated it.

Should I update `test_remoteAddr()` to do the same as well?

Reply:
Thanks. I think it would help to also update `test_remoteAddr`.

Let me know once this PR is ready for re-review.

Thanks again for the update

Reply:
Thanks @adiroiban, I've also applied the same update to `test_remoteAddr`